proto/v2/skaffold.proto

```diff
@@ -181,9 +181,10 @@ message ApplicationLogEvent {
 message TaskEvent {
     string id = 1; // will be used by SkaffoldLog to link it to a task. Follows the form "{task_name}-{iteration-number}"
     string task = 2; // task name oneof: Tag, Build, Test, Deploy, StatusCheck, PortForward, DevLoop
-    int32 iteration = 3; // which dev/debug iteration is currently running
-    string status = 4; // artifact build status oneof: InProgress, Completed, Failed
-    ActionableErr actionableErr = 5; // actionable error message
+    string text = 3; // additional more descriptive text for the task
```

Comment:
Should we rename it to description?
```suggestion
    string description = 3; // additional more descriptive text for the task
```

Reply:
Thanks, wasn't sure what the best name would be 👍🏼